app/policies/workspace_policy.rb

```diff
@@ -0,0 +1,7 @@
+class WorkspacePolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+end
```

Comment:
here they also have a lot more - but just ignore it if im wrong ;) I was just a bit confused with the pundit topic - thats why i checked the lecture.
![image](https://user-images.githubusercontent.com/55313319/68709663-7686cb00-0596-11ea-86d6-ff6695e0eebf.png)

Reply:
with pundit , steps taken 
1. added the gem / bundle install/rails g pundit:install
2. Pundit setup (2) copied into the application controller
3.rails g pundit:policy workspace / booking done 

4. authorize @booking / @workspace  in set_booking and set_work space 

nothing else done , will then go through the lecture  just doing the workspace but authorization in its own separate branch

Reply:
i have not changed anything in application policy

Reply:
nice